sklearn/neural_network/_multilayer_perceptron.py

```diff
@@ -178,7 +179,7 @@ def _forward_pass(self, activations):
 
         return activations
 
-    def _forward_pass_fast(self, X):
+    def _forward_pass_fast(self, X, validation=True):
```

Comment:
In other parts of the library, we frequently use the name `check_input` for this functionality:

```suggestion
    def _forward_pass_fast(self, X, check_input=True):
```